src/app.html

```diff
@@ -125,6 +125,25 @@
       }
     </script>
 
+    <!-- Init menu as fast as possible as well -->
+    <script>
+      try {
+        const currentMenu =
+          localStorage.nnsMenu === null ||
```

Comment:
nitpick: Just because it's a pure js, how about `!localStorage.nnsMenu ? undefined ...`? Just to preserve some bits.

Reply:
Good idea! I copied the code we have for the `localstorage.nnsTheme` but, that's a good idea, let's do it.

Reply:
Done.